test/source/tests/settings.ts

```diff
@@ -770,7 +770,9 @@ export const defineSettingsTests = (testVariant: TestVariant, testWithBrowser: T
         await myKeyPage.waitAndType('@input-prv-key', keyCanBeFixedTestKey);
         await myKeyPage.waitAndType('@input-passphrase', 'hard to guess passphrase');
         await myKeyPage.waitAndClick('@action-update-key');
-        await myKeyPage.waitAndRespondToModal('confirm', 'confirm', 'Public and private key updated locally');
+        await myKeyPage.waitAll('@input-compatibility-fix-expire-years', { timeout: 30 });
+        await myKeyPage.selectOption('@input-compatibility-fix-expire-years', '1');
+        await myKeyPage.waitAndClick('@action-fix-and-import-key');
```

Comment:
let's also check that key was successfully fixed, as currently we just click `fix` button and test will pass even if some error will be shown after this click. so we should also check imported key

Reply:
I have added a simple check for the expiry date ensuring that its value was updated correctly as expected.